public/index.html

```diff
@@ -0,0 +1,741 @@
+                <li>
+                  <figure class="container-icon"><p class="icon-pin"></p></figure>
+                  <div class="container-icon-caption">
+                    <h1 class="container-icon-title">Address</h1>
+                    <p>121 King Street, Melbourne<br>Victoria, Australia 3000</p>
+                  </div>
+                </li>
+                <li>
+                  <figure class="container-icon"><p class="icon-phone"></p></figure>
+                  <div class="container-icon-caption">
+                    <h1 class="container-icon-title">Free Helpline</h1>
+                    <p>(001) 800 345 333700<br>(002) 600 345 333700</p>
+                  </div>
+                </li>
+                <li>
+                  <figure class="container-icon"><p class="icon-email"></p></figure>
+                  <div class="container-icon-caption">
+                    <h1 class="container-icon-title">Email</h1>
+                    <p>info@appbox.com<br>feedback@appbox.com</p>
+                  </div>
+                </li>
+              </ul>
+              <section class="form-section">
+                <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce.Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce.</p>
+                <form class="contact-form">
+                  <div class="contact-form-first-line">
+                    <input type="text" name="fullname" placeholder="Full Name">
+                    <input type="email" name="email" placeholder="Email">
+                  </div>
+                  <textarea rows="4" cols="50" placeholder="Message"></textarea>
+                </form>
+                <input class="contact-form-submit-button" type="submit" value="SEND">
+              </section>
+            </article>
+          </section>
+          <a href="home"></a>
+          <footer class="f3-reviews-footer">
+            <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+              <path d="M0 0 L0 100 L100 0 Z" />
+            </svg>
+            <figure class="footer-logo"><img src="./img/appbox-v2.png" alt="AppBox Logo"></figure>
+            <div class="copyright"><p>Copyright &#64; 2015. Designed by Templines</p></div>
+            <ul class="social-media">
+              <li class="icon-facebook icon-social-media"></li>
+              <li class="icon-twitter icon-social-media"></li>
+              <li class="icon-google-plus icon-social-media"></li>
+              <li class="icon-pinterest icon-social-media"></li>
+              <li class="icon-behance icon-social-media"></li>
+              <li class="icon-dribbble icon-social-media"></li>
+            </ul>
+          </footer>
+          <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+          <script type="text/javascript" src="js/main.js"></script>
```

Comment:
You're going to want this to be the last file you load, even after your plugins (if anything you write is dependent on a plugin, it's going to cause trouble).